sans/reduction.ipynb

```diff
@@ -210,6 +210,55 @@
     "    return (pixel_size * pixel_length) / (L2 * L2)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def q_resolution(lam_edges, moderator, d):\n",
+    "    moderator = sc.rebin(moderator, 'wavelength', lam_edges)\n",
+    "    \n",
+    "    d_lam = lam_edges['wavelength', 1:] - lam_edges['wavelength', :-1] # bin widths\n",
+    "    lam = 0.5 * (lam_edges['wavelength', 1:] + lam_edges['wavelength', :-1]) # bin centres\n",
+    "    \n",
+    "    r2 = sc.Variable(value=4.0824829046386295/1000, unit=sc.units.m) # sample aperture radius\n",
+    "    r1 = sc.Variable(value=14.433756729740645/1000, unit=sc.units.m) # source aperture radius\n",
+    "    dr = sc.Variable(value=8.0/1000, unit=sc.units.m) # virtual ring width on detector\n",
+    "    \n",
+    "    l2 = sc.neutron.l2(d)\n",
+    "    l1 = sc.Variable(value=5.0, unit=sc.units.m) # should be sc.neutron.l1(d) = 25.6m ?\n",
```

Comment:
I think this is just a misleading naming in the equation, it should *not* be l1, but the collimation length, as specified for the algorithm. We *do* need the actualy L1 for the moderator calculation though (but see the snippet I had suggested to avoid this altorgether).

Reply:
I'll change the name and remove my comment to avoid confusion.